### PR TITLE
Added Disk Inventory X

### PR DESCRIPTION
**Pull request by dutzi**
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in alphabetical order
- [x] Appropriate language icon(s) added if applicable
- [ ] Screenshots(s) added if any
- [ ] Has a commit from less than 2 years ago
- [ ] Has a **clear** README in English

### CODE REVIEW

**Comment by dutzi** [comment]
Oh, crap, I just realized it isn't open source...
Still free and pretty useful though

File: README.md [hunk 1]
```diff
@@ -525,6 +525,7 @@ Mac app wrapper around Google's stand-alone Android Messenger. ![javascript_icon
 - [Clean-Me](https://github.com/Kevin-De-Koninck/Clean-Me) - Small macOS app that acts as a system cleaner (logs, cache, ...). ![swift_icon]
 - [Diagnostics](https://github.com/macmade/Diagnostics) - Diagnostics is an application displaying the diagnostic reports from applications on macOS. ![swift_icon]
 - [DisableMonitor](https://github.com/Eun/DisableMonitor) - Easily disable or enable a monitor on your Mac. ![objective_c_icon]
+- [Disk Inventory X](http://www.derlien.com/) - Shows the sizes of files and folders in a special graphical way called "treemaps". ![objective_c_icon]
```
Comment (serhii-londar) [inline README.md]: HI @dutzi, can you change website URL to git URL?

Comment (dutzi) [reply README.md]: I wish, turns out it's not open source 😕

Comment (serhii-londar) [reply README.md]: I'm sorry but i can't merge it( I'll close this PR.

**Pull request closed without merging**